src/ResourceManager/ApplicationInsights/Commands.ApplicationInsights/Commands.Storage.Netcore.csproj

```diff
@@ -0,0 +1,66 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@frankguodongchen please remove this file

Reply:
removed, thanks!